include/dca/phys/dca_step/cluster_solver/shared_tools/accumulation/tp/tp_accumulator.hpp

```diff
@@ -101,6 +101,9 @@ class TpAccumulator<Parameters, linalg::CPU> {
   // Empty method for compatibility with GPU version.
   void finalize() {}
 
+  // Empty method for compatibility with GPU version.
+  void ringG() {}
```

Comment:
conceivably this stub could be useful to nodes with limited memory versus G4 as well. Nothing need go here at least until the GPU version works well.

Reply:
Right, @efdazedo also suggested having a cpu version of ringG algorithm. As long as gpu version looks good to the team, I can write one.